### DISCUSSION TITLE
How Can I add "40" for meeting or calls?

### DISCUSSION THREAD

**Discussion by OscarGS86** [YetiForceCompany/YetiForceCRM#15381] [How to] [answered]
Hi guys,

All my meetings are for 40 minutes by default. How I can add 40 here in the dropdown to make my meeting for 40 minutes by default in the calendar?

![Screenshot 2021-10-28 at 17 55 41](https://user-images.githubusercontent.com/92228499/139292525-a09de4f3-8505-473f-9b66-ffa202242bb4.png)

Cheers,

**Comment by vovpff** [comment]
I think you can add it manually in database

**Reply by OscarGS86** [reply]
Thanks @vovpff ,

Do you know in which table I can find those values to add a new one?

Cheers

**Reply by vovpff** [reply]
No. You can find it by yourself

**Comment by rskrzypczak** [comment, marked as answer]
Here: 
https://github.com/YetiForceCompany/YetiForceCRM/blob/a755bb518da22746789f2750f4957b601f265b11/modules/Users/uitypes/MultiDependField.php#L28

**Reply by OscarGS86** [reply]
Thank you @rskrzypczak !

Sorted!

Cheers
